deploy.go

```diff
@@ -53,9 +58,10 @@ var flags = []cli.Flag{
 		Usage: "The git ref to deploy. Can be a git commit, branch or tag.",
 	},
 	cli.StringFlag{
-		Name:  "env, e",
-		Value: "",
-		Usage: "The environment to deploy to.",
+		Name:        "env, e",
+		Value:       "",
+		Usage:       "The environment to deploy to.",
+		Destination: &env,
```

Comment:
Eh.. not much. Just saving an extra `env := c.String("env")` here: https://github.com/remind101/deploy/blob/alias-environments/deploy.go#L150

I can go either way, but I don't believe that `Destination:` is harmful